lib/util/string_utils.hpp

```diff
@@ -18,6 +18,15 @@ namespace oklt::util {
  */
 std::string toLower(const std::string& str);
 
+/**
+ * @brief check a string starts with prefix value
+ *
+ * @param str Input string
+ * @param str Prefix string
+ * @return bool Is started with provided prefix
+ */
+bool starts_with(const std::string& value, const std::string& prefix);
```

Comment:
Please rename it to `startsWith` as per the coding conventions that we agreed upon.